content/de/cloud-native-tech.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Cloud Native Technologien
+status: Feedback erwünscht
```

Comment:
The EN term "[Cloud Native Technology](https://glossary.cncf.io/cloud-native-tech/)" is GA, so let's mark this one "Completed". 😊

```suggestion
status: Completed
```

Reply:
@jihoon-seo adjusted :)